---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

## Education

**present, PhD student in Computational Biology**.  Institute for Quantitative Biology, Biochemistry and Biotechnology, The University of Edinburgh. Edinburgh, UK. Supervised by Dr. G. Stracquadanio and Prof. G. Sanguinetti

<ins>Research Interests</ins>: cancer heritability estimation from GWAS data; deep neural networks for network inference; gene expression analysis.

**2017, MSc. in Biomedical Engineering**. Università degli Studi di Pisa, Italy.  
 
<ins>Dissertation</ins>: “Classification of resting state fMRI datasets: machine learning methods for the identification of patients with anxiety disorders”. Supervisors: N. Vanello, L. Citi, C. Gentili.  

<ins>Relevant modules</ins> : stochastic processes, signal processing, bioinformatics, statistical analysis, exploratory data analysis, database development and programming, digital and analogue electronics.       

**2017, BSc. in Biomedical Engineering**. Università degli Studi di Pisa, Italy.   

## Recent publications 

 * Fanfani, V. and Stracquadanio, G. ‘A unified framework for geneset network analysis’, bioRxiv,   2019, doi: 10.1101/699926
 * Fanfani, V.  Citi, L., Harris, AL, Pezzella, F. and Stracquadanio, G. ‘Gene-level heritability analysis explains the polygenic architecture of cancer’. bioRxiv, 2019, doi: 10.1101/599753
 
## Conferences

* 2019, Genes and Cancer Meeting, Cambridge, UK. Selected flash talk: "Dissecting cancer heritability in European populations".  
* 2018, From functional genomics to system biology, EMBL, Heidelberg, DL. Poster presentation: "Geneset Network Analysis: understanding high-throughput genomic data using the interactome".  
* 2018, BACR Students Conference.  Poster presentation: “A new geneset analysis approach to identify and characterise cancer pathways”   

## Recent Talks
 
 * 2019 – “Cancer heritability, decoding the contribution of high frequency variants to cancer risk”. 
            SynthSys, University of Edinburgh 

  
## Skills

* __Core Competencies__: Algebra, number theory, and calculus. Univariate and multivariate statistics. Bayesian inference. Graphs. Stochastic signal processing and image processing. Machine learning. Chemistry, biochemistry, and physiology fundamentals.
* __Programming__ 
  * Languages: python, R, MATLAB & Simulink, C/C++, SQL, android
  * Scientific packages: numpy, scipy, pymc, pytorch, networkx
  * Workflow systems : snakemake, nextflow
  * Data visualisation, Python and R
* __Other__: Git, Latex, Photoshop, Illustrator

## Teaching

* 2019 – Tutor on Next Generation Sequencing for "Tools for Synthetic Biology". 
 The University of Edinburgh.   
* 2019 – Invited instructor. "Analytics & Data Science Summer School 2019”.   
  IADS, University of Essex, UK. Course title: "Learning from Small Data"
* 2018 – Teaching and Lab Assistant at the University of Essex for the PGR introductory course 
  "CSEE Bootcamp".   
  Fundamentals of data analysis and research methods.
* 2017 - 2018 –  Graduate Laboratory Assistant at the University of Essex, for the following courses:
  * "Data Science and Decision Making"
  * "Foundations of Electronics II"
  * "Introduction to Databases"
  * "Introduction to programming" 


## Selected awards

 * **2019 – James Rennie Bequest**. Travel award. The Biology of Genomes, Cold Spring Harbor 	 Laboratory, US.   
 * **2019 - NVidia Data science GPU program**. Nominated research software engineering developer.  
 * **2016 – Erasmus+ scholarship**. Six months traineeship at the University of Essex.   

## Memberships of professional bodies
Italian State Exam for Qualified Information Engineer, 2018

## Languages
* English, Proficient.
* German, Intermediate
* French, Spanish Basics
* Italian, native speaker
