---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

## Experience

**Postdoctoral Researcher**. Harvard School of Public Health, Department of Biostatistics. Boston, MA. 
  Topics and interests:
      - Inference of individual-specific Gene Regulatory Networks for personalized medicine
      - Analysis of multi-omics assays to infer tumor evolution trajectories.
      - Software and workflow development
      - Teaching and mentoring 

**Research Technician**. The University of Edinburgh. Edinburgh UK
  - Analysis of sequencing data for synthetic biology
  - Graph Neural Networks for interpretable cancer network modeling and driver gene identification

## Education

**PhD student in Computational Biology**.  Institute for Quantitative Biology, Biochemistry and Biotechnology, The University of Edinburgh. Edinburgh, UK. Supervised by Dr. G. Stracquadanio and Prof. G. Sanguinetti

<ins>Research Interests</ins>: cancer heritability estimation from GWAS data; deep neural networks for network inference; gene expression analysis.

**2017, MSc. in Biomedical Engineering**. Università degli Studi di Pisa, Italy.  
 
<ins>Dissertation</ins>: “Classification of resting state fMRI datasets: machine learning methods for the identification of patients with anxiety disorders”. Supervisors: N. Vanello, L. Citi, C. Gentili.  

<ins>Relevant modules</ins> : stochastic processes, signal processing, bioinformatics, statistical analysis, exploratory data analysis, database development and programming, digital and analogue electronics.       

**2017, BSc. in Biomedical Engineering**. Università degli Studi di Pisa, Italy.   

## Recent publications 
Full list is on [Google Scholar](https://scholar.google.co.uk/citations?user=rAJMiSMAAAAJ&hl=en)

 * Saha, E. & Fanfani, V. et al. 'Bayesian inference of sample-specific coexpression networks'. Genome Research, 2024. https://doi.org/10.1101/gr.279117.124
 * Hossein, I. et al. 'Biologically informed NeuralODEs for genome-wide regulatory dynamics'. Genome Biology, 2024. https://doi.org/10.1186/s13059-024-03264-0 
 * Ben Guebila, M. et al. 'The Network Zoo: a multilingual package for the inference and analysis of gene regulatory
  networks'. Genome Biology 2023. https://doi.org/10.1186/s13059-023-02877-1
 * Zhao et al. 'Debugging and consolidating multiple synthetic chromosomes reveals combinatorial genetic interactions'. Cell, 2023 
 * Fanfani et al. 'Discovering cancer driver genes and pathways using stochastic block model graph neural networks'. Biorxiv 2021
 * Fanfani, Viola, et al. ‘The landscape of the heritable cancer genome‘. Cancer Research, 2021. https://cancerres.aacrjournals.org/content/early/2021/03/12/0008-5472.CAN-20-3348
 * Fanfani, Viola, et al. ‘PyGNA: a unified framework for geneset network analysis‘. BMC Bioinformatics, 2020. https://doi.org/10.1186/s12859-020-03801-1
 * Fanfani, Viola, et al. ‘Dissecting the heritable risk of breast cancer: from statistical methods to susceptibility genes.’ Seminars in Cancer Biology. Academic Press, 2020. https://doi.org/10.1016/j.semcancer.2020.06.001
 * Draberova, Helena, et al. ‘Systematic analysis of the IL‐17 receptor signalosome reveals a robust regulatory feedback loop.’ The EMBO Journal (2020): e104202. doi: https://doi.org/10.15252/embj.2019104202
 
## Conferences
* 2024, RECOMB, Cambridge, US. Presentation: “BONOBO: Bayesian Optimized sample-specific Networks Obtained By Omics data".
* 2022, Reproducibility, Replicability and Trust in Science (virtual), Wellcome Genome Campus, UK. "Research practices, reproducibility, and overhead in biostatistics and bioinformatics"
* 2021, Network Biology virtual meeting, CSHL, US. Poster Presentation: “Interpretable graph neural networks unveil
  system‑level reprogramming in cancer”.
* 2020, The Biology of Genomes virtual meeting, CSHL, US. Poster Presentation: "Decoding cancer risk in the broader population with gene-level heritability".
* 2019, Genes and Cancer Meeting, Cambridge, UK. Selected flash talk: "Dissecting cancer heritability in European populations".  
* 2018, From functional genomics to system biology, EMBL, Heidelberg, DL. Poster presentation: "Geneset Network Analysis: understanding high-throughput genomic data using the interactome".  
* 2018, BACR Students Conference.  Poster presentation: “A new geneset analysis approach to identify and characterise cancer pathways”   

## Other
 
 * 2019 – Invited talk “Cancer heritability, decoding the contribution of high frequency variants to cancer risk”. 
            SynthSys, University of Edinburgh 
 * I have served as reviewer for BMC Bioinformatics, BMC Genomics, and Frontiers in Molecular Biosciences.

  
## Skills

* __Core Competencies__: Algebra, number theory, and calculus. Univariate and multivariate statistics. Bayesian inference. Graphs. Stochastic signal processing and image processing. Machine learning. Chemistry, biochemistry, and physiology fundamentals.
* __Programming__ 
  * Languages: python, R, MATLAB & Simulink, C/C++, SQL, android/Java
  * Scientific packages: numpy, scipy, pymc, pytorch, networkx
  * Workflow systems : nextflow, snakemake
  * Data visualisation, Python and R, Photoshop, Illustrator
* __Other__: Git, Latex, docker, conda, AWS, bash, HPC

##  Maintained Software

 * __netZooPy__. Python package for gene regulatory network methods https://github.com/netZoo/netZooPy
 * __BAGHERA__.  Bayesian Gene Heritability Analysis from GWAS summary statistics. https://github.com/stracquadaniolab/baghera
 * __PyGNA__. A python package for Geneset Network Analysis. https://github.com/stracquadaniolab/pygna


## Teaching
* 2023 & 2024 - Instructor. "BST270, Reproducible Data Science", Harvard School of Public Health, Boston, US  
* 2021 - Invited Instructor "Primer on Bayesian modelling", SynBio Society, Edinburgh, UK.  
* 2019 & 2020 – Tutor on Next Generation Sequencing for "Tools for Synthetic Biology". 
 The University of Edinburgh.   
* 2019 & 2021 – Invited instructor. "Analytics & Data Science Summer School 2019”.   
  IADS, University of Essex, UK. Course title: "Learning from Small Data"
* 2018 – Teaching and Lab Assistant at the University of Essex for the PGR introductory course 
  "CSEE Bootcamp".   
  Fundamentals of data analysis and research methods.
* 2017 - 2018 –  Graduate Laboratory Assistant at the University of Essex, for the following courses:
  * "Data Science and Decision Making"
  * "Foundations of Electronics II"
  * "Introduction to Databases"
  * "Introduction to programming" 


## Selected awards

 * **2019 – James Rennie Bequest**. Travel award. The Biology of Genomes, Cold Spring Harbor 	 Laboratory, US.   
 * **2019 - NVidia Data science GPU program**. Nominated research software engineering developer.  
 * **2016 – Erasmus+ scholarship**. Six months traineeship at the University of Essex.   

## Memberships of professional bodies
Italian State Exam for Qualified Information Engineer, 2018

## Languages
* English, Proficient.
* German, Intermediate
* French, Spanish Basics
* Italian, native speaker
