---
layout: archive
title: "Research"
permalink: /research/
author_profile: true
---

## Publications

 * Saha, E. & Fanfani, V. et al. _'Bayesian inference of sample-specific coexpression networks'_. Genome Research, 2024. https://doi.org/10.1101/gr.279117.124
 * Hossein, I. et al. _'Biologically informed NeuralODEs for genome-wide regulatory dynamics'_. Genome Biology, 2024. https://doi.org/10.1186/s13059-024-03264-0 
 * Ben Guebila, M. et al. _'The Network Zoo: a multilingual package for the inference and analysis of gene regulatory
  networks'_. Genome Biology 2023. https://doi.org/10.1186/s13059-023-02877-1
 * Zhao et al. _'Debugging and consolidating multiple synthetic chromosomes reveals combinatorial genetic interactions'_. Cell, 2023 
 * Fanfani et al. _'Discovering cancer driver genes and pathways using stochastic block model graph neural networks'_. Biorxiv 2021
 * Fanfani, Viola, et al. _‘The landscape of the heritable cancer genome‘_. Cancer Research, 2021. doi: [10.1158/0008-5472.CAN-20-3348](https://cancerres.aacrjournals.org/content/early/2021/03/12/0008-5472.CAN-20-3348)
 * Fanfani, Viola, et al. _‘PyGNA: a unified framework for geneset network analysis.‘_ BMC Bioinformatics, 2020. doi: [https://doi.org/10.1186/s12859-020-03801-1](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03801-1)
 * Fanfani, Viola, et al. _‘Dissecting the heritable risk of breast cancer: from statistical methods to susceptibility genes.‘_ Seminars in Cancer Biology. Academic Press, 2020. doi: [https://doi.org/10.1016/j.semcancer.2020.06.001](https://doi.org/10.1016/j.semcancer.2020.06.001)
 * Draberova, Helena, et al. _‘Systematic analysis of the IL‐17 receptor signalosome reveals a robust regulatory feedback loop.’_ The EMBO Journal (2020): e104202. doi: [https://doi.org/10.15252/embj.2019104202](https://doi.org/10.15252/embj.2019104202)
 * Fanfani, V.  Citi, L., Harris, AL, Pezzella, F. and Stracquadanio, G. _‘Gene-level heritability analysis explains the polygenic architecture of cancer’_. bioRxiv, 2019, doi: 10.1101/599753

## Conferences
* 2024, __RECOMB__, Cambridge, US. Presentation: “BONOBO: Bayesian Optimized sample-specific Networks Obtained By Omics data".
* 2022, __Reproducibility, Replicability and Trust in Science__(virtual), Wellcome Genome Campus, UK. "Research practices, reproducibility, and overhead in biostatistics and bioinformatics"
 * 2021, __"Network Biology"__ virtual meeting, CSHL, US. _Poster Presentation_: “Interpretable graph neural networks unveil system‑level reprogramming in cancer”.
 * 2020, __"The Biology of Genomes"__, virtual meeting, CSHL, US. _Poster Presentation_: "Decoding cancer risk in the broader population with gene-level heritability".
 * 2019, __Genes and Cancer Meeting__, Cambridge, UK. _Selected flash talk_: "Dissecting cancer heritability in European populations".  
 * 2018, __From functional genomics to system biology__, EMBL, Heidelberg, DL. _Poster presentation_: "Geneset Network Analysis: understanding high-throughput genomic data using the interactome".  
 * 2018, __BACR Students Conference__.  _Poster presentation_: “A new geneset analysis approach to identify and characterise cancer pathways”    

## Recent Talks

 * __2019 – SynthSys, University of Edinburgh__ , “Cancer heritability, decoding the contribution of high frequency variants to cancer risk”. 
            

# Projects

On GitHub you can see all the projects I am working on. Most of them are hosted on the 
 [Quackenbush Lab](https://github.com/QuackenbushLab "Github") and
 [Stracquadanio Lab](https://github.com/stracquadaniolab "Github").

###  <img src="../images/netZoo_logo_v14.png" alt="drawing" width="200"/>
<br />
The Network Zoo, Python version ([netZooPy](https://github.com/netZoo/netZooPy "netZooPy")).

###  <img src="../images/logo.png" alt="drawing" width="200"/>
<br />
The Bayesian Gene Heritability Analysis software ([BAGHERA](https://github.com/stracquadaniolab/baghera "BAGHERA")) estimates the contribution to the heritability of a trait/disease of all the SNPs in the genome (genome-wide heritability) and those nearby protein-coding genes (gene-level heritability).

Documentation can be found [here](https://baghera.readthedocs.io/en/latest/ "Baghera").

###  <img src="../images/pygna_logo.png" alt="drawing" width="150"/>
<br />
[PyGNA](https://github.com/stracquadaniolab/pygna "PyGNA") is a unified framework for network analysis of high-throughput experiment results. It can be used both as a standalone command line application or it can be included as a package in your own python code.

Pygna has now been published in BMC Bioinformatics! [https://doi.org/10.1186/s12859-020-03801-1](https://doi.org/10.1186/s12859-020-03801-1)

Documentation can be found [here](https://pygna.readthedocs.io/en/latest/ "Pygna").
