---
layout: archive
title: "Research"
permalink: /research/
author_profile: true
---

## Publications

 * Fanfani, V. and Stracquadanio, G. _‘A unified framework for geneset network analysis’_, bioRxiv,   2019, doi: 10.1101/699926
 * Fanfani, V.  Citi, L., Harris, AL, Pezzella, F. and Stracquadanio, G. _‘Gene-level heritability analysis explains the polygenic architecture of cancer’_. bioRxiv, 2019, doi: 10.1101/599753

## Conferences

 * 2019, __Genes and Cancer Meeting__, Cambridge, UK. _Selected flash talk_: "Dissecting cancer heritability in European populations".  
 * 2018, __From functional genomics to system biology__, EMBL, Heidelberg, DL. _Poster presentation_: "Geneset Network Analysis: understanding high-throughput genomic data using the interactome".  
 * 2018, __BACR Students Conference__.  _Poster presentation_: “A new geneset analysis approach to identify and characterise cancer pathways”    

## Recent Talks

 * __2019 – SynthSys, University of Edinburgh__ , “Cancer heritability, decoding the contribution of high frequency variants to cancer risk”. 
            

# Projects

On GitHub you can see all the projects I am working on. Most of them are hosted on the lab's 
 [lab's group](https://github.com/stracquadaniolab "Github").

###  <img src="../images/logo.png" alt="drawing" width="200"/>
<br />
The Bayesian Gene Heritability Analysis software ([BAGHERA](https://github.com/stracquadaniolab/baghera "BAGHERA")) estimates the contribution to the heritability of a trait/disease of all the SNPs in the genome (genome-wide heritability) and those nearby protein-coding genes (gene-level heritability).

Documentation can be found [here](https://baghera.readthedocs.io/en/latest/ "Baghera").

###  <img src="../images/pygna_logo.png" alt="drawing" width="150"/>
<br />
[PyGNA](https://github.com/stracquadaniolab/pygna "PyGNA") is a unified framework for network analysis of high-throughput experiment results. It can be used both as a standalone command line application or it can be included as a package in your own python code.

Documentation can be found [here](https://pygna.readthedocs.io/en/latest/ "Pygna").